src/stratis_cli/_actions/_debug.py

```diff
@@ -40,6 +42,25 @@ def refresh_state(_namespace):
         if return_code != StratisdErrors.OK:  # pragma: no cover
             raise StratisCliEngineError(return_code, message)
 
+    @staticmethod
+    def send_uevent(_namespace):
+        """
+        Issue a synthetic uevent from the CLI.
+        """
+        try:  # pragma: no cover
+            realdevice = os.path.realpath(_namespace.device, strict=True)
+            sysfs_device_uevent = (
+                realdevice.replace("/dev/", "/sys/class/block/") + "/uevent"
+            )
+            uevent_str = "change"
```

Comment:
Might as well drop the assignment here and just use the literal string ```"change"``` in the function call below.

Reply:
Resolved via force-push